web/src/components/TestCard/TestCard.tsx

```diff
@@ -39,12 +39,16 @@ const TestCard: React.FC<TTestCardProps> = ({
 
   return (
     <S.TestCard $isCollapsed={isCollapsed}>
-      <S.InfoContainer>
-        {isCollapsed ? (
-          <DownOutlined onClick={() => setIsCollapsed(false)} />
-        ) : (
-          <RightOutlined data-cy={`collapse-test-${testId}`} onClick={onCollapse} />
-        )}
+      <S.InfoContainer
+        onClick={async () => {
+          if (isCollapsed) {
+            setIsCollapsed(false);
+            return;
+          }
+          await onCollapse();
```

Comment:
I don't think we need the await here

Reply:
onCollapse return a promise.
I can either await or void it